src/tools/miri/tests/fail/panic/double_panic.rs

```diff
@@ -1,6 +1,5 @@
-//@error-pattern: the program aborted
+//@error-pattern: panic in a function that cannot unwind
 //@normalize-stderr-test: "\| +\^+" -> "| ^"
-//@normalize-stderr-test: "unsafe \{ libc::abort\(\) \}|crate::intrinsics::abort\(\);" -> "ABORT();"
 
 struct Foo;
 impl Drop for Foo {
```

Comment:
Could you also add a Miri test for the new code  that should work now (in the MR description)?

Reply:
Done.